vespa/package.py

```diff
@@ -1415,6 +1417,57 @@ def load(disk_folder: str) -> "ApplicationPackage":
         with open(file_path, "r") as f:
             return ApplicationPackage.from_json(f.read())
 
+    def to_zip(self) -> BytesIO:
```

Comment:
My understanding is that we will unify not only exporting to zip for both Docker and Cloud deployment but also the `VespaDocker.export_application_files` method, otherwise we will have code duplication that will be hard to support going forward.

Reply:
Yes I am working on the export, too. In the transition it will be duplication in order to not break too much / there is so much code to change. When done and no tests fail, we can look at notebooks and change these - then do final cleanup